repo/meta/schema/v3-resource-schema.json

```diff
@@ -68,11 +76,13 @@
             }
           },
           "additionalProperties": false,
-          "type": "object"
+          "type": "object",
+          "description": "This is a mapping of aliases to docker images. Aliases enumerated here are available during resolution of marathon.json.mustache file."
```

Comment:
Suggest: `This is a mapping of aliases to Docker images. The aliases enumerated here are available during resolution of marathon.json.mustache file.`